import-automation/progress-dashboard-rest/app/service/base_database.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Base class for a Database service that stores some kind of entities using
+Google Cloud Datastore for storage.
+"""
+
+from google.cloud import datastore
+
+from app import utils
+
+
+class BaseDatabase:
+    """Base class for a Database service that stores some kind of entities using
+    Google Cloud Datastore for storage.
+
+    New Entities must be created using get_by_id(make_new=True).
```

Comment:
Is "Entities" a class?  If not, should it be lowercased?

Reply:
Entity is a datastore class